apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -291,8 +298,7 @@ export default function SectionsSetUpContainer({sectionToBeEdited}) {
           onClick={e => {
             e.preventDefault();
             setIsSaveInProgress(true);
-            recordSectionSetupEvent(sections[0]);
```

Comment:
I moved this call into the save function right before the redirect to ensure the event is a true successful save